src/sync/installation.ts

```diff
@@ -63,8 +64,23 @@ export const sortedRepos = (repos: Repositories): [string, RepositoryData][] =>
 			new Date(a[1].repository?.updated_at || 0).getTime()
 	);
 
-const getNextTask = async (subscription: Subscription): Promise<Task | undefined> => {
-	const tasks = taskTypes;
+export const getTargetTasks = async (jiraHost: string, allTasks: TaskType[], targetTasks?: TaskType[]): Promise<TaskType[]> => {
```

Comment:
`allTasks` doesn't need to exist since we have access to the allTaskTypes global, one extra argument to make it purer was a small cost.